usr/share/rear/conf/default.conf

```diff
@@ -2264,7 +2266,8 @@ GALAXY11_Q_ARGUMENTFILE=
 # CommVault login credentials for restore
 # Remember to adequately protect the rescue media if you include credentials in it
 GALAXY11_USER=${GALAXY11_USER:-}
-GALAXY11_PASSWORD=${GALAXY11_PASSWORD:-}
+{ GALAXY11_PASSWORD=${GALAXY11_PASSWORD:-} ; } 2>/dev/null
+# In local.conf set it confidentially via { GALAXY11_PASSWORD='secret_password' ; } 2>/dev/null
```

Comment:
I have this special additional explanation because
```
{ GALAXY11_PASSWORD=${GALAXY11_PASSWORD:-} ; } 2>/dev/null
```
is not a correct template how to set it in local.conf but
```
{ GALAXY11_PASSWORD='secret_password' ; } 2>/dev/null
```
is a template which can be used even for copy&paste.

Reply:
I'd like to leave at least a little bit of own thinking to our users, so having a general explanation for how to set secret variables at the top of `default.conf` should really suffice. For my taste, we already have way too many long explanations in our `default.conf` that I need to scroll by while looking for the actual variables.

Reply:
My experience over almost 25 years with various kind of users
tells that the majority of users who think about what they do
is not the problem - the problem are the few who do not think.

Nevertheless - hope dies last - so:
https://github.com/rear/rear/pull/2982/commits/1d8d0e8270626ae25270ef01237fe6b888417acd